src/appengine/private/components/testcase-detail/testcase-detail.html

```diff
@@ -596,6 +597,8 @@
               testcase-id="[[info.testcase.id]]"></crash-stats>
         </template>
 
+        <testcase-variants info="[[info]]"></testcase-variants>
```

Comment:
only testcase id is used here. better to just pass testcase-id="[[info.testcase.id]]" ?

Reply:
Done!